templates/templates/base.html

```diff
@@ -13,9 +13,14 @@
   <link rel="apple-touch-icon-precomposed" sizes="72x72" href="{{ STATIC_URL }}img/apple-touch-icon-72x72-precomposed.png">
   <link rel="apple-touch-icon-precomposed" href="{{ STATIC_URL }}img/apple-touch-icon-precomposed.png">
   <link type="text/plain" rel="author" href="{% versioned_static 'files/humans.txt' %}" />
-  <link rel="stylesheet" type="text/css" media="screen" href="{% versioned_static 'css/styles.css' %}" />
+  <link rel="stylesheet" type="text/css" media="screen" href="{% versioned_static 'css/styles.min.css' %}" />
+  
+
+  <!-- global nav -->
+  <script async src="{{ STATIC_URL }}components/canonical-global-nav/build/js/global.js"></script>
```

Comment:
This should be removed.

Reply:
Have to keep it, if remove it the top navi bar will gone.